lib/mumuki/domain/status.rb

```diff
@@ -1,6 +1,8 @@
 module Mumuki::Domain::Status
   extend ActiveSupport::Concern
 
+  include Comparable
```

Comment:
I like the idea of having ..::Skipped > ..::Failed but 
a) I'm not sure if it's semantically meaningful
b) It could lead to some unexpected behavior:
The status has a custom implementation of `==` but if it didn't this would cause `Skipped == Passed` as it would compare them based on the exp points.

I think your first implementation is a bit more declarative. Maybe `tops?` or `improves?` are good alternative as well.

Reply:
Just to be clear: I love the fact that the comparison is done based on the exp_points. I would implement the method the same way, I'd just use something custom instead of `>`

Reply:
Alright. I first did `lesser_than?` as you saw but I hated the name. I'll revert this commit and change the original implementation to `improves?`.